CONTRIBUTING.md

```diff
@@ -51,20 +51,14 @@ Please mail to secure-opensource@intel.com directly for security issue
 
 ## Public issue tracking
 
-If you have a problem, please let us know.  IRC is a perfectly fine place
-to quickly informally bring something up, if you get a response.  The
-[mailing list](https://lists.01.org/postorius/lists/intel-vaapi-media.lists.01.org)
-is a more durable communication channel.
-
 If it's a bug not already documented, by all means please [open an
 issue in github](https://github.com/intel/libva/issues/new) so we all get visibility
 to the problem and can work towards a resolution.
 
 For feature requests we're also using github issues, with the label
 "enhancement".
 
-Our github bug/enhancement backlog and work queue are tracked in a
-[Libva waffle.io kanban](https://waffle.io/intel/libva).
+If you have a problem, and is unsuitable to file issue, or is urgent to get support, you could send a mail to maintainers in https://github.com/orgs/intel/teams/libva-maintainers/members
```

Comment:
I am unsure that this link is visible by non intel and/or non-members.: if I am not signed in and click on it, github ask me to sign in explicitly. Can someone check whether it's actually accessible? @evelikov : can you, please, help?

Reply:
That's correct - people outside of the (Intel?) team cannot open the URL.

Here's my understanding and links to another project for illustration:
Github shows people part of a namespace/org (assuming those people didn't set themselves as hidden) - for example see the "People" section in https://github.com/mpv-player/ and the complete list in https://github.com/orgs/mpv-player/people. AFAICT Github do not allow listing of team members for a given project within the namespace - there is no People section in https://github.com/mpv-player/mpv

Aside: this MR is out of date - the ML and waffle references got removed a week or two ago.